update_prevalence.py

```diff
@@ -1205,6 +1205,8 @@ def ignore_jhu_place(line: JHUCommonFields) -> bool:
         "Micronesia",
         "Palau",
         "Summer Olympics 2020",
+        "Winter Olympics 2022",
```

Comment:
Note that there are a handful of things like this that may pop up over time, which could result in a failed run until we update the blocklist here.  I'd prefer to get a better failed-run notification system in place before merging this PR to mitigate this risk.  If along the way we can get a concept of a 'warning' that becomes equally visible, but discards the locale without failing the run, perhaps there would be room to make this a warning instead.